src/PolyhedralGeometry/PolyhedralComplex/properties.jl

```diff
@@ -65,8 +65,6 @@ end
 
 vertices(as::Type{Union{RayVector{T}, PointVector{T}}}, PC::PolyhedralComplex) where T<:scalar_types = SubObjectIterator{as}(pm_object(PC), _vertex_or_ray_polyhedron, length(_all_vertex_indices(pm_object(PC))))
 
-vertices(::Type{Union{RayVector, PointVector}}, PC::PolyhedralComplex) where T<:scalar_types = vertices(Union{RayVector{T}, PointVector{T}}, PC)
```

Comment:
This line makes no sense because `T` is not defined on the right side, so I expect this would run into an error when it would get executed. According to our [code coverage report](https://app.codecov.io/gh/oscar-system/Oscar.jl/blob/master/src/PolyhedralGeometry/PolyhedralComplex/properties.jl), it is not executed by our tests.

This lines comes from commit 1eb480a63d65b358498e48f9b62c85ae3e80c975 from PR #1020 . As far as I can tell, it replaced this line:
```julia
vertices(::Type{Union{RayVector, PointVector}}, PC::PolyhedralComplex) = vertices(Union{RayVector{Polymake.Rational}, PointVector{Polymake.Rational}}, PC)
```

So perhaps the correct solution is to not remove it but to replace it, but I couldn't tell. If we replace it, we should have a test for it.